config/custom-environment-variables.yaml

```diff
@@ -30,6 +30,16 @@ executor:
             nodeSelectors:
               __name: K8S_NODE_SELECTORS
               __format: json
+            # k8s preferred node selectors for try to approprate build pod scheduling
```

Comment:
Can probably reduce the comment to say
```
# k8s preferred node selectors for build pod scheduling
# See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature
```
(since `nodeSelectors` is pretty much the same thing and has the full description)

Reply:
fixed!